### PR TITLE
Rename props `isFocused` to `focused` in <Label />

### PR DESCRIPTION
**Pull request by Andresbl123**
The `isFocused` property is renamed to `focused` in the <Label />component. This change makes the code easier to read and conforms to the design system standards.

**BREAKING CHANGE**
This name change may cause the component to not behave as expected in the places where it is used. It is important to validate and **adjust the code so that the component** works correctly with the new property name.

### CODE REVIEW

File: src/components/inputs/Label/index.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const defaultTypo = "labelLarge";
 const Label = (props: ILabelProps) => {
   const {
     isDisabled = defaultIsDisabled,
-    isFocused = defaultIsFocused,
+    focused = defaultIsFocused,
```
Comment (cmarin001) [inline src/components/inputs/Label/index.tsx:22]: sigue existiendo el isFocused

File: src/components/inputs/Label/index.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ const Label = (props: ILabelProps) => {
     typeof isDisabled === "boolean" ? isDisabled : defaultIsDisabled;
 
   const transformedIsFocused =
-    typeof isFocused === "boolean" ? isFocused : defaultIsFocused;
+    typeof focused === "boolean" ? focused : defaultIsFocused;
```
Comment (cmarin001) [inline src/components/inputs/Label/index.tsx:33]: sigue existiendo el isFocused

File: src/components/inputs/Label/index.tsx [hunk 3]
```diff
@@ -40,7 +40,7 @@ const Label = (props: ILabelProps) => {
   return (
     <StyledLabel
       isDisabled={transformedIsDisabled}
-      isFocused={transformedIsFocused}
+      focused={transformedIsFocused}
```
Comment (cmarin001) [inline src/components/inputs/Label/index.tsx:43]: sigue existiendo el isFocused

**Review by cmarin001 — COMMENTED** [review]
revisar todos los componentes donde se usa <Label/>
image

File: src/components/inputs/Label/index.tsx (hunk 1 above)
Comment (Andresbl123) [reply src/components/inputs/Label/index.tsx:22]: `defaultIsFocused` no se modifica ya que en la tarjeta ids416 estos valores son eleminados ya que no son necesario, y asi evitar posibles conflictos

**Review by cmarin001 — APPROVED** [review]
cumple con los requerimientos

**Pull request merged**